geode-core/src/main/java/org/apache/geode/internal/security/IntegratedSecurityService.java

```diff
@@ -272,6 +260,11 @@ public void authorize(ResourcePermission context, Subject currentUser) {
       return;
     }
 
+    Object principal = currentUser.getPrincipal();
+    if (securityManager.isExpired(principal)) {
+      throw new AuthenticationExpiredException(principal + " expired.");
```

Comment:
Given that `Object.toString()` is used to more diagnostic purposes does it make sense to include that in the message? It may result information leak about the credential we don't want serialized. Simply throwing this exception should be information enough.